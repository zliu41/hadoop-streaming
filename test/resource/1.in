(3,8)
(3,9)
(3,10)
(4,5)
(4,6)
(5,10)
