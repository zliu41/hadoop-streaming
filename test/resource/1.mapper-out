1003, 108
1003, 109
1003, 110
1004, 105
1004, 106
1005, 110
