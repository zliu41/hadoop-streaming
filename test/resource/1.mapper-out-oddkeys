1003, 108
1003, 109
1003, 110
1005, 110
