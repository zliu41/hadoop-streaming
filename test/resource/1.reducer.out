3, 27
4, 11
5, 10
